packages/lit-element/src/lib/updating-element.ts

```diff
@@ -422,13 +422,17 @@ export abstract class UpdatingElement extends HTMLElement {
     );
   }
 
+  protected static get hasFinalized() {
+    return this.hasOwnProperty(finalized);
```

Comment:
I'd consider making this a helper function rather than a method. It'll be more likely to get inlined by terser.

Reply:
Removed.